qemu/tests/cfg/rng_maxbytes_period.cfg

```diff
@@ -0,0 +1,46 @@
+- rng_maxbytes_period:
+    type = rng_maxbytes_period
+    kill_vm = yes
+    virt_test_type = qemu
+    no no_virtio_rng
+    only Linux
+    read_rng_cmd  = "dd if=/dev/hwrng of=/dev/null bs=1024 count=1024"
+    check_rngd_service = "systemctl status rngd"
+    start_rngd_service = "systemctl start rngd"
+    RHEL.6:
+        driver_name = virtio
+        check_rngd_service = "service rngd status"
+        start_rngd_service = "service rngd start"
+    variants:
+        - positive_test:
+            variants:
+                - scenario1:
+                    max-bytes_virtio-rng-pci = 512
```

Comment:
the mix use '-' and '_' could not be good, usually, we use '_' for a param definition.

Reply:
This is the restriction in the avocado-vt, please check function 'add_virtio_rng' in file 'qemu_vm.py'.
The prefix 'max-bytes'  is the option name of virtio-rng-pci device in qemu command line, and 'virtio-rng-pci' is the required suffix in function 'add_virtio_rng'.
Thanks!